src/main/java/com/gradle/enterprise/conventions/customvalueprovider/ScanCustomValueNames.java

```diff
@@ -1,10 +1,10 @@
 package com.gradle.enterprise.conventions.customvalueprovider;
 
 public interface ScanCustomValueNames {
-    String GIT_COMMIT_NAME = "Git Commit ID";
-    String GIT_STATUS = "Git Status";
-    String GIT_BRANCH_NAME = "Git Branch Name";
-    String BUILD_ID = "Build ID";
-    String IDEA_VERSION = "IDEA version";
+    String GIT_COMMIT_NAME = "Git_Commit_ID";
```

Comment:
I am not sure the `_` is better than the ` `. I think we should either use a human readable description or some case convention. Camel case or snake case are both fine. For example we could use `gitCommitId` or `git_commit_id`. Mixing case within the parts of the name doesn't seem to be a good idea for entering those keys in the search box.

Reply:
given that we already have `watchFs`, let's stick in camel case.

Reply:
Camel case sounds good to me.

Reply:
Done, PTAL!